### PR TITLE
Add Nordpool Sensor documentation

### PR DESCRIPTION
**Pull request by MTrab**
**Description:**
The Nordpool Elspot market is the market for electrical power in the Scandinavian and Baltic countries.
This sensor fetches the current price for electrical power and put it into a sensor.

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by MTrab** [comment]
Erhm?! Why is my Repetier pull mixed in with this?
It's a completely different branch?!

File: source/_components/sensor.nordpool.markdown [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+title: "Nordpool Sensor"
+description: "Instructions on how to integrate Nordpool within Home Assistant."
+date: 2019-03-06
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: nordpool.png
+ha_category: Energy
+ha_release: 0.88
```
Comment (klaasnicolaas) [inline source/_components/sensor.nordpool.markdown]:
```suggestion
ha_release: 0.90
```

File: source/_components/sensor.nordpool.markdown [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+{% configuration %} repetier:
+  type: list
+  required: true
+  keys:
+    name:
+      description: Name of the sensor.
+      required: true
+      type: string
+    currency:
+      description: Your preferred currency. Valid currencies are: DKK, NOK, SEK, EUR (case-sensitive)
```
Comment (klaasnicolaas) [inline source/_components/sensor.nordpool.markdown]:
```suggestion
      description: "Your preferred currency. Valid currencies are: DKK, NOK, SEK, EUR (case-sensitive)"
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by MTrab** [comment]
Closing this - unfortunately Nordpool are requiring a 1900EUR yearly license pr. user for this, and are not willing to negotiate.

**Pull request closed without merging**